Source/API/CBLJSON.m

```diff
@@ -125,7 +125,11 @@ + (size_t) estimateMemorySize: (id)object {
         // Thanks to DenNukem's answer in http://stackoverflow.com/questions/399527/
         sFormatter = [[NSDateFormatter alloc] init];
         sFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
+#ifdef __IPHONE_8_0
+        sFormatter.calendar = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
```

Comment:
There's no need to #ifdef this — NSCalendarIdentifierGregorian is available back to iOS 4. Just change the constant.